internal/datastore/discussion_access.go

```diff
@@ -226,21 +107,32 @@ func (iter *discussionIter) Next(discussion *model.Discussion) bool {
 		return false
 	}
 
+	titleHistory := make([]byte, 0)
```

Comment:
Does this work from a JSON Unmarshal standpoint? Seems like we'd need an empty dictionary

Reply:
it works because of the code you added for resolving the values

![image](https://user-images.githubusercontent.com/65272234/89114941-974dc400-d436-11ea-81bc-d64fd57972bf.png)